frontend/packages/knative-plugin/src/catalog/event-source-data.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  EventSourceApiServerModel,
+  EventSourceContainerModel,
+  EventSourceKafkaModel,
+  EventSourcePingModel,
+  EventSourceSinkBindingModel,
+} from '../models';
+
+const eventSourceCatalogProviderData: {
+  [key: string]: { documentationUrl?: string; description?: string; provider?: string };
+} = {
+  [EventSourceApiServerModel.kind]: {
+    documentationUrl:
+      'https://docs.openshift.com/container-platform/4.6/serverless/event_sources/serverless-apiserversource.html',
+    description:
+      'This object can be used to connect an event sink, such as a service, channel, or broker to the Kubernetes API server. ApiServerSource watches for Kubernetes events and forwards them to the sink.',
```

Comment:
localize

Reply:
Do we localize provider as well? `t('Red Hat')`